### ISSUE TITLE
Scrum Report Sprint 1 (May 11 - 25)

### ISSUE THREAD

**Issue by achin797** [achin797/tagify#1]

**Comment by achin797** [comment]
**What I did:**
• Created repo
• Suggested the idea
• Submitted the tag page sketch

**Major challenges I/we ran into**
• Deciding on the topic to work on took a few minutes
• Sketching was hard since I'm not good at drawing

**What I plan to do**
• Register the app with Spotify
• Setup up basic server and client for authentication

**Comment by akvaale** [comment]
**What did you work on this past iteration (2 weeks)?**
I worked on the planning of requirements, and listing those requirements from most important to least important. I also sketched up the log-in page design.

**What were any major issues/challenges you ran into?**
Designing and sketching the website itself was a unique challenge that I had never faced before. I had to look up other web apps to gain inspiration

**What do you plan to work on for this coming iteration (2 weeks)?**
-Research capabilities of Spotify API
-Start planning on the organization of Data (for the tags)
-Research React UI libraries

**Comment by mingyugao** [comment]
What did I work on?
I worked on contributing to shaping the project idea and suggesting some technical details.

What were any major issues/challenges I ran into?
Breaking the idea down into specific minimal, standard and stretch requirements was challenging, especially considering that the idea might change over the next few weeks.

What do I plan to work on for this coming iteration?
- Set up repo with meteor and react/redux stuff + git branches
- Research Spotify API
- Research React UI libraries
